src/generated/resources/.cache/2fac88980105fe60a243d9c77aac6e4aa28502c1

```diff
@@ -0,0 +1 @@
+// 1.20.2	2023-12-10T19:03:07.9528872	Tags for minecraft:item mod id dysonsphereproject
```

Comment:
IDK what this .cache is but we might need to gitignore it

Reply:
looks like mekanism keeps it in